scripts/upgrade/sonarr.sh

```diff
@@ -1,8 +0,0 @@
-#!/bin/bash
-
-if [[ ! -f /install/.sonarr.lock ]]; then
-    echo_error "Sonarr not detected. Exiting!"
-    exit 1
-fi
-
-box install sonarrv3
```

Comment:
This upgrader should stay. It should instead check for `sonarrv2-old` and install `sonarr`